test/es6module/otherModule.js

```diff
@@ -4,5 +4,5 @@
 //-------------------------------------------------------------------------------------------------------
 
 //test that dynamic import works from module
-//this file is imported only once - dynamicly from another module to check that the host is notified for that
```

Comment:
As per above comment, this is an unrelated correction to one of my previous commits.